<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Widget</class>
 <widget class="QWidget" name="Widget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>545</width>
    <height>466</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>545</width>
    <height>466</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>545</width>
    <height>466</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Calcul du poids idéal</string>
  </property>
  <widget class="QTabWidget" name="tabWidget">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>30</y>
     <width>441</width>
     <height>371</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(255, 255, 255);</string>
   </property>
   <property name="currentIndex">
    <number>0</number>
   </property>
   <widget class="QWidget" name="tab">
    <attribute name="title">
     <string>Informations personelles</string>
    </attribute>
    <widget class="QPushButton" name="pushButtonSuite">
     <property name="geometry">
      <rect>
       <x>190</x>
       <y>260</y>
       <width>80</width>
       <height>25</height>
      </rect>
     </property>
     <property name="text">
      <string>Suite</string>
     </property>
    </widget>
    <widget class="QWidget" name="">
     <property name="geometry">
      <rect>
       <x>140</x>
       <y>160</y>
       <width>181</width>
       <height>28</height>
      </rect>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QLabel" name="labelAge">
        <property name="text">
         <string>Age</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QSpinBox" name="spinBoxAge">
        <property name="minimum">
         <number>1</number>
        </property>
        <property name="maximum">
         <number>200</number>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
    <widget class="QWidget" name="">
     <property name="geometry">
      <rect>
       <x>140</x>
       <y>190</y>
       <width>181</width>
       <height>28</height>
      </rect>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <widget class="QLabel" name="labelPoids">
        <property name="text">
         <string>Poids (kg)</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QDoubleSpinBox" name="doubleSpinBoxPoids">
        <property name="minimum">
         <double>15.000000000000000</double>
        </property>
        <property name="maximum">
         <double>300.000000000000000</double>
        </property>
        <property name="singleStep">
         <double>0.100000000000000</double>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
    <widget class="QWidget" name="">
     <property name="geometry">
      <rect>
       <x>140</x>
       <y>220</y>
       <width>181</width>
       <height>28</height>
      </rect>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_3">
      <item>
       <widget class="QLabel" name="labelTaille">
        <property name="text">
         <string>Taille (m)</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QDoubleSpinBox" name="doubleSpinBoxTaille">
        <property name="minimum">
         <double>0.500000000000000</double>
        </property>
        <property name="maximum">
         <double>3.000000000000000</double>
        </property>
        <property name="singleStep">
         <double>0.010000000000000</double>
        </property>
        <property name="value">
         <double>0.500000000000000</double>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
    <widget class="QWidget" name="">
     <property name="geometry">
      <rect>
       <x>140</x>
       <y>130</y>
       <width>180</width>
       <height>25</height>
      </rect>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_4">
      <item>
       <widget class="QLabel" name="labelSexe">
        <property name="text">
         <string>Sexe</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="radioButtonSexeF">
        <property name="text">
         <string>Femme</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="radioButtonSexeH">
        <property name="text">
         <string>Homme</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
    <widget class="QWidget" name="">
     <property name="geometry">
      <rect>
       <x>140</x>
       <y>100</y>
       <width>181</width>
       <height>27</height>
      </rect>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_5">
      <item>
       <widget class="QLabel" name="labelPrenom">
        <property name="text">
         <string>Prénom</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="lineEditPrenom">
        <property name="clearButtonEnabled">
         <bool>true</bool>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
    <widget class="QWidget" name="">
     <property name="geometry">
      <rect>
       <x>140</x>
       <y>70</y>
       <width>181</width>
       <height>27</height>
      </rect>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_6">
      <item>
       <widget class="QLabel" name="labelNom">
        <property name="text">
         <string>Nom</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="lineEditNom">
        <property name="inputMethodHints">
         <set>Qt::ImhNone</set>
        </property>
        <property name="inputMask">
         <string/>
        </property>
        <property name="dragEnabled">
         <bool>false</bool>
        </property>
        <property name="readOnly">
         <bool>false</bool>
        </property>
        <property name="clearButtonEnabled">
         <bool>true</bool>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </widget>
   <widget class="QWidget" name="tab_2">
    <attribute name="title">
     <string>Calcul du poids idéal</string>
    </attribute>
    <widget class="QTextEdit" name="textEditAffichage">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>411</width>
       <height>201</height>
      </rect>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButtonDevine">
     <property name="geometry">
      <rect>
       <x>70</x>
       <y>240</y>
       <width>301</width>
       <height>25</height>
      </rect>
     </property>
     <property name="text">
      <string>Poids idéal formule de devine</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButtonLorentz">
     <property name="geometry">
      <rect>
       <x>70</x>
       <y>270</y>
       <width>301</width>
       <height>25</height>
      </rect>
     </property>
     <property name="text">
      <string>Poids idéal formule de lorentz</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButtonLorentzAge">
     <property name="geometry">
      <rect>
       <x>70</x>
       <y>300</y>
       <width>301</width>
       <height>25</height>
      </rect>
     </property>
     <property name="text">
      <string>Poids idéal formule de lorentz en fonction de l'age</string>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QPushButton" name="pushButtonQuitter">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>420</y>
     <width>80</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>Quitter</string>
   </property>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections>
  <connection>
   <sender>pushButtonQuitter</sender>
   <signal>clicked()</signal>
   <receiver>Widget</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>275</x>
     <y>429</y>
    </hint>
    <hint type="destinationlabel">
     <x>203</x>
     <y>434</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
